### ISSUE TITLE
Failed to upload file in two ways probably because of file size

### ISSUE THREAD

**Issue by qin-yu** [bioimage-io/archive.bioimage.io#357]
- When the model and other data are compressed into one ZIP file
  - bioimage.io says: `Failed to upload file: Error: Bug : uncompressed data size mismatch`
- When submit the RDF file first and upload other files directly
  - bioimage.io says: `Failed to upload file: NotReadableError: The requested file could not be read, typically due to permission problems that have occurred after a reference to a file was acquired.`

Note that these two attempts failed when the test input has size `180 x 180 x 180` and test output of my model is 33 dimensions of that (~2.x GB, a StarDist output). Reducing test input size by half lead to a successful upload. 

Is this expected or not?

**Comment by esgomezm** [comment]
Hi @qin-yu 

Thank you for the issue. Probably you're referring to [this model](https://github.com/bioimage-io/collection-bioimage-io/pull/646). I'm checking the image (test_image.npy) uploaded to [zenodo](https://zenodo.org/record/8421756) and it has shape 96x96x96. However, I can see in the model specifications that the input has a shape 116x116x66. Could you please provide an image of the same shape as the input of the model (116x116x66)?

Also could you please confirm that the error was in the uploading interface please?

**Comment by qin-yu** [comment]
Hi @esgomezm Thanks for checking! 

> Also could you please confirm that the error was in the uploading interface please?

Yes, the errors happened during uploading.

> Could you please provide an image of the same shape as the input of the model (116x116x66)?

The whole ZIP package was generated using StarDist's built-in export-to-bioimage.io function, so I assumed that they must have figured out the size. But if it doesn't look correct to you, I'll see how I can fix it.

**Comment by qin-yu** [comment]
Hi @esgomezm I wonder how should I update the mode? If I just drop the ZIP file into bioimage.io upload interface, it will give me a new DOI. Is this expected?

**Comment by esgomezm** [comment]
Yes, so the way to go is that we blocked the "wrong" version of the model in the PR, merge that PR and when you upload again, using the same zenodo account, it will suggest you update the model. Please, if it doesn't suggest you to update the model, let us know.
